NSS-Day-Cohort-55/tabloidfullstack-adequately-powerful-ibexes
# Lj add tag

# Description
User can  create a tag
Fixes # (34)


# Testing Instructions
fetch and checkout to branch lj-addTag
-log in to app
-click "Tag Management" in the navbar
-click the "Add A Tag" button
-give your new tag a name and click the "Save" button
-confirm that your new tag is now in the tag list
